### PR TITLE
OSDOCS-9849: adds 4.15.9 relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15

Issue:
[OSDOCS-9849](https://issues.redhat.com/browse/OSDOCS-9849)

Link to docs preview:
[4.15.9 async release note](https://74550--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes.html#microshift-4-15-9-dp)

QE review:
- N/A release notes stock text

### CODE REVIEW

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -269,4 +269,13 @@ Issued: 2024-04-08
 
 {product-title} release 4.15.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1670[RHBA-2024:1670] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1668[RHSA-2024:1668] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-15-9-dp"]
+=== RHBA-2024:1772 - {microshift-short} 4.15.9 bug fix and security update advisory
+
+Issued: 2024-04-16
```
Comment (lpettyjo) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:277]: OK that date is in the future?

**Review by lpettyjo — COMMENTED** [review]
LGTM! Just one comment.

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 2]
```diff
@@ -269,4 +269,13 @@ Issued: 2024-04-08
 
 {product-title} release 4.15.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1670[RHBA-2024:1670] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1668[RHSA-2024:1668] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```
Comment (lpettyjo) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:272]: First two links generate 404 errors. I assume that this content will be available when this is release?

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: microshift_release_notes/microshift-4-15-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:277]: This PR  will be merged only after the RHBA is shipped live on 16th April. So should be good.

File: microshift_release_notes/microshift-4-15-release-notes.adoc (hunk 2 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:272]: Yes I think so.

**Pull request merged**